lib/msal-angular/src/packageMetadata.ts

```diff
@@ -1,3 +1,3 @@
 /* eslint-disable header/header */
-export const name = '@azure/msal-angular';
-export const version = '3.0.7';
+export const name = "@azure/msal-angular";
```

Comment:
Hector mentioned last week that we have to change this back to single quotes for the format check to pass